troposphere/static/js/components/admin/ResourceRequest/ResourceRequest.jsx

```diff
@@ -62,17 +62,24 @@ export default React.createClass({
 
     onIdentitySave(identity) {
         let { selectedRequest: request } = this.props;
+        let username = request.get("created_by").username;
```

Comment:
Nice. 👏

Reply:
Ha - nevermind. The `let { selectedRequest: request } ...` was already there. But 👏 anyway.